docs/guide/troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+# Troubleshooting
+
+See [rollup's troubleshooting](https://rollupjs.org/guide/en/#troubleshooting) too.
+
+If you didn't make it work, please try posting questions on [GitHub Discussions](https://github.com/vitejs/vite/discussions) or discussioning the issue in #help channel on [Vite Land Discord](https://chat.vitejs.dev).
+
+## CLI
+
+### `Error: Cannot find module 'C:\foo\bar&baz\vite\bin\vite.js'`
+
+Probably the path to your project folder includes `?`.
+There is a bug in `npm` when using Windows and the path including `?` ([npm/cmd-shim#45](https://github.com/npm/cmd-shim/issues/45)).
+
+You will need to either:
+
+- Switch to other package manager (e.g. `pnpm`, `yarn`)
+- Remove `?` from the path to your project
+
+## Dev Server
+
+### Requests are stalled forever
+
+If you are using linux, file descriptor limits and inotify limits may be affecting.
+Because Vite does not bundle most of the files, browsers requests many files and this requires many file descriptors.
+
+To solve this:
+
+- Increase file descriptor limits by `ulimit`
+- Increase the following inotify related limits by `sysctl`
+  - `fs.inotify.max_queued_events`
+  - `fs.inotify.max_user_instances`
+  - `fs.inotify.max_user_watches`
```

Comment:
I think we could use some code examples here. I think https://github.com/vitejs/vite/issues/5310#issuecomment-986042734 and https://github.com/vitejs/vite/issues/5310#issuecomment-1181860523 would do. Something like:

```bash
# Check current ulimit
$ ulimit -Sn

# Change limit
$ ulimit -Sn 10000
```

Make sure to restart your browser after changing the limit.

---

```bash
# Check current fs.inotify.* values
$ sysctl fs.inotify

# Change a fs.inotify.* value
$ sudo sysctl fs.inotify.max_user_instances=8192
```

Reply:
Done 👍